### PR TITLE
Editorial feedback from readers on the style chapter

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: src/lab6.py [hunk 1]
```diff
@@ -249,6 +249,14 @@ def paint(self):
                                          word, font, color))
         return cmds
 
+    @wbetools.js_hide
```
Comment (chrishtr) [inline src/lab6.py:252]: These methods were removed in this chapter due to a refactor; this seems a reasonable way to get rid of them from the outline.

File: Makefile [hunk 2]
```diff
@@ -69,7 +69,7 @@ latex/book.pdf: latex/book.tex latex/macros.tex
 	(cd latex && ln -f -s ../www/im/ im)
 	(cd latex && pdflatex book.tex)
 
-www/%.html: book/%.md infra/template.html infra/signup.html infra/filter.lua config.json
+www/%.html: book/%.md infra/template.html infra/signup.html infra/filter.lua config.json src/lab*.py
```
Comment (pavpanchekha) [inline Makefile:72]: What does this do? Also, maybe nothing?

File: src/lab6.py (hunk 1 above)
Comment (pavpanchekha) [reply src/lab6.py:252]: Yeah it would be nice to have something nice but this is definitely a good fix for now.

**Review by pavpanchekha — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (chrishtr) [reply Makefile:72]: The book chapters depend on the source code for the outline. This addition expresses that for this particular rule.

**Pull request merged**